log/logger.go

```diff
@@ -10,3 +10,59 @@ type Formatable interface {
 	String() string
 	JSON() string
 }
+
+// DefaultLogger ...
+type DefaultLogger struct {
+	ts bool
+}
+
+type logfunc func(string, ...interface{})
+
+// Timestamp ...
+func (dl DefaultLogger) Timestamp() DefaultLogger {
```

Comment:
we may want to enable and later disable the timestamp logging for the same logger instance.